schemas/book.schema.tpl.json

```diff
@@ -0,0 +1,17 @@
+{
+  "_type": "https://openminds.ebrains.eu/publications/Book",
+  "_extends": "creativeWork.schema.tpl.json",
+  "required": [
+    "name",
```

Comment:
should the "digitalIdentifier" not also be a required field?

Reply:
looking up ISBNs of books when all you want to do is cite one chapter from the book is a tedious business, so I'd prefer to make it optional.

Reply:
okay